src/core/agent/services/connectionService.ts

```diff
@@ -80,27 +97,28 @@ class ConnectionService extends AgentService {
         isMultiSigInvite: multiSigInvite,
         connectionId: undefined,
         status: ConnectionStatus.PENDING,
+        url,
       },
     });
 
-    const operation = await this.resolveOobi(url, multiSigInvite);
-    const connectionId =
-      operation.done && operation.response
-        ? operation.response.i
-        : new URL(url).pathname.split("/oobi/").pop()?.split("/")[0];
+    const connectionId = new URL(url).pathname
+      .split("/oobi/")
+      .pop()!
+      .split("/")[0];
+    const alias = new URL(url).searchParams.get("name") ?? uuidv4();
```

Comment:
we are moving from using uuid to salter - check with Martin. uuid isn't actually a dependency we have so we shouldn't be calling it, it's a sub dependency (not sure why JS allows this)